src/lang/Messages.properties

```diff
@@ -1849,7 +1849,8 @@ pscan.options.dialog.scanner.field.label.enabled                 = Enabled:
 pscan.options.dialog.scanner.field.label.name                    = Name:
 pscan.options.dialog.scanner.field.label.type                    = Type:
 pscan.options.main.name = Passive Scanner
-pscan.options.main.label.scanOnlyInScope = Only scan messages in scope:
+pscan.options.main.label.scanOnlyInScope = Only scan messages in scope
+pscan.options.main.label.scanFuzzerMessages = Include traffic from the Fuzzer when passive scanning 
```

Comment:
The space at the end could be removed (although it does not make much difference).